umbra/umbra_32.c

```diff
@@ -531,6 +572,19 @@ umbra_arch_exit()
 drmf_status_t
 umbra_map_arch_init(umbra_map_t *map, umbra_map_options_t *ops)
 {
+    /* Override redzone size. We must make it a page size to control
+     * its permissions.
+     *
+     * FIXME i#2283: Potentially support larger redzone sizes to be even more safe.
```

Comment:
nit: s/FIXME/XXX/
(not a correctness issue)

Reply:
Done.